icons/nuget/nuget-original-wordmark.svg

```diff
@@ -0,0 +1 @@
fill-opacity="1"><path d="M.723 1.5c-.664.777-.75 1.125-.637 2.047.203 1.473.953 2.223 2.422 2.426.922.113 1.27.027 2.047-.637.777-.664.922-.953.922-2.047C5.477 1.5 4.555.58 2.766.58 1.672.578 1.383.722.723 1.5zm0 0M12.684 7.125c-1.817.664-4.352 3.258-4.985 5.105-.695 2.02-.722 13.56-.031 15.547.578 1.73 2.422 3.782 4.266 4.762l1.472.777H27.82l1.442-.777c1.73-.894 2.996-2.223 3.922-3.98.66-1.27.691-1.5.691-8.512v-7.211l-.781-1.441c-.977-1.875-2.305-3.118-4.266-4.012-1.555-.719-1.7-.75-8.215-.719-5.277 0-6.89.113-7.93.461zm3.402 4.355c1.644 1.528 1.586 3.75-.113 5.223-2.598 2.277-6.489-.203-5.567-3.578.664-2.395 3.864-3.316 5.68-1.645zm11.617 8.512c3.922 2.074 3.922 7.844 0 9.922-3.285 1.73-7.176-.059-8.098-3.691-1.156-4.618 3.86-8.453 8.098-6.23zm0 0M122.61 8.973c-.606.23-.665.43-.665 2.422v2.164h-1.586c-1.5 0-1.586.027-1.586.718 0 .696.086.723 1.586.723h1.586v6.492c0 6.98.172 7.988 1.473 8.91.777.52 3.168.637 4.035.176.29-.176.547-.578.547-.894 0-.52-.117-.579-.777-.375-1.328.375-2.422.23-3.059-.434-.605-.605-.633-.895-.719-7.242L123.36 15h2.305c2.309 0 2.336 0 2.336-.723 0-.718-.027-.718-2.305-.718h-2.277l-.09-2.426c-.086-2.363-.086-2.39-.719-2.16zm0 0M53.188 13.79c-.633.288-1.583 1.007-2.102 1.612l-.922 1.07V15c0-1.414-.031-1.441-.867-1.441h-.863v17.02h1.73v-5.712c0-5.02.086-5.828.574-6.84 1.125-2.394 2.54-3.316 4.961-3.316 1.442 0 1.785.117 2.649.894.578.493 1.156 1.356 1.328 1.934.144.578.289 3.75.289 7.04v6h1.73v-5.626c0-7.473-.433-9.262-2.683-10.933-1.125-.836-4.239-.954-5.825-.23zm0 0M88.82 13.672c-3.543 1.27-5.765 6.086-4.93 10.762.376 2.164.868 3.316 1.993 4.5 1.21 1.355 2.824 2.02 4.785 2.046 2.277 0 3.719-.52 5.074-1.875l1.121-1.125v2.25c0 4.56-2.363 6.98-6.8 6.98-1.415 0-4.008-.663-4.586-1.183-.461-.46-.72-.285-.72.493 0 1.183 2.075 1.96 5.333 1.988 2.969 0 4.351-.403 5.793-1.672 2.367-2.047 2.71-3.922 2.71-14.828v-8.45h-.863c-.835 0-.867.06-.867 1.212v1.21l-.922-.98c-.52-.52-1.355-1.125-1.87-1.328-1.126-.488-3.923-.488-5.25 0zm4.961 1.355c.922.493 2.219 1.73 2.68 2.625.289.551.402 1.82.402 4.27 0 2.945-.086 3.664-.574 4.644-1.586 3.118-6.055 4.098-8.535 1.907-1.5-1.301-2.192-2.887-2.363-5.282-.23-3.718.98-6.664 3.316-7.875 1.238-.664 4.063-.808 5.074-.289zm0 0M107.703 13.703c-1.383.488-3.168 2.192-3.95 3.777-1.929 4.012-.862 10.098 2.134 12.262 2.191 1.586 5.765 1.7 8.765.23 1.04-.492 1.239-.75 1.239-1.413v-.836l-1.641.863c-3.172 1.617-5.996 1.387-7.93-.633-1.093-1.156-1.96-3.348-1.96-4.933v-.81h12.742l-.2-1.815c-.55-4.79-2.71-7.153-6.547-7.125-.835.027-2.015.203-2.652.433zm5.883 2.363c1.066 1.125 1.73 2.684 1.73 4.04v.664H104.36l.176-.81c.547-2.335 1.785-4.093 3.43-4.843.547-.262 1.555-.375 2.71-.32 1.727.086 1.903.176 2.91 1.27zm0 0M66.363 19.96c.086 5.282.203 6.638.637 7.587 1.035 2.426 2.621 3.433 5.36 3.433 2.163 0 3.23-.46 4.585-1.988l.89-1.039v1.324c0 1.243.032 1.301.868 1.301h.863v-17.02h-1.73v5.97c0 5.48-.055 6.058-.602 7.242-1.472 3.027-5.68 3.863-7.816 1.558-1.207-1.3-1.383-2.336-1.383-8.742v-6.027H66.25zm0 0"/></g></svg>
```

Comment:
Please remove all that whitespace below the icon.

![image](https://user-images.githubusercontent.com/51391473/141689977-83d0de1f-5c74-487c-a0d0-98abe5788dfe.png)

Reply:
Wide icons were centered instead of changing the viewBox, was this what you meant by removing whitespaces?
+ <a href="https://github.com/devicons/devicon/blob/master/icons/haskell/haskell-original-wordmark.svg">Haskell</a>
+ <a href="https://github.com/devicons/devicon/blob/master/icons/express/express-original-wordmark.svg">Express</a>
+ <a href="https://github.com/devicons/devicon/blob/master/icons/knockout/knockout-plain-wordmark.svg">Knockout</a>

Reply:
> Wide icons in the repo were centered vertically, was this what you meant by removing whitespaces?
> 
> * [Haskell](https://github.com/devicons/devicon/blob/master/icons/haskell/haskell-original-wordmark.svg)
> * [Express](https://github.com/devicons/devicon/blob/master/icons/express/express-original-wordmark.svg)
> * [Knockout](https://github.com/devicons/devicon/blob/master/icons/knockout/knockout-plain-wordmark.svg)

I meant to remove all of that blank space that the icon has, it just makes the logo bigger and it's unnecessary.
Those icons you mentioned *are* on the center, which makes it look good, though.

Reply:
This is what I could get done, I don't really know how I can remove the whitespaces and make this a rectangular logo instead of a square without changing the viewBox. (In the documentation it is stated that the viewBox should be "0 0 128 128")
![example](https://user-images.githubusercontent.com/54141068/141795472-ae9c4030-7a13-46a1-85e5-18dd71812ded.png)

Reply:
> This is what I could get done, I don't really know how I can remove the whitespaces and make this a rectangular logo instead of a square without changing the viewBox. (In the documentation it is stated that the viewBox should be "0 0 128 128") ![example](https://user-images.githubusercontent.com/54141068/141795472-ae9c4030-7a13-46a1-85e5-18dd71812ded.png)

That looks awesome! Thanks. 😄